src/Model/MailingList/testClient.java

```diff
@@ -0,0 +1,21 @@
+package Model.MailingList;
+
+
+
+/**
+ * @author Jordan
+ * demo showing how to notify observers
+ */
+public class testClient {
```

Comment:
This class is good for testing purposes, but test code should be placed in scr/test/java source folder as unit test class.